Queues/JobQueue.cs

```diff
@@ -213,7 +214,8 @@ public AlgorithmNodePacket NextJob(out string location)
                 Language = Language,
                 Parameters = parameters,
                 Controls = controls,
-                PythonVirtualEnvironment = Config.Get("python-venv")
+                PythonVirtualEnvironment = Config.Get("python-venv"),
+                HostName = Environment.MachineName
```

Comment:
We don't really need these changes I think? given the baseResulthandler is already getting MachineName if hostname isn't set?